app/main/views/services.py

```diff
@@ -201,11 +193,7 @@ def import_service(service_id):
 
     db.session.add(service)
 
-    try:
-        db.session.commit()
-    except IntegrityError:
-        db.session.rollback()
-        abort(400, "Unknown supplier ID provided")
```

Comment:
We should have a test to check that when a service is added with an unknown supplier_id the response code is 400